apps/package.json

```diff
@@ -139,12 +117,14 @@
     "node-sass": "1.2.2",
     "npm-which": "2.0.0",
     "phantomjs-prebuilt": "^2.1.3",
-    "playground-io": "bcjordan/playground-io#more-components",
+    "playground-io": "bcjordan/playground-io#tap-cap-touch",
```

Comment:
Confirmed, this line was previously non-functional and should have been removed during development (was accidentally left here during package extraction), and wound up referencing an older version.